assets/src/components/propertiesPanel/vehiclePropertiesPanel.tsx

```diff
@@ -198,15 +199,25 @@ const StatusContent = ({ selectedVehicle }: { selectedVehicle: Vehicle }) => (
   </>
 )
 
-const VehiclePropertiesPanel = ({ selectedVehicle }: Props) => {
-  const [tabMode, setTabMode] = useState<TabMode>("status")
+const VehiclePropertiesPanel = ({
+  selectedVehicle,
+  initialTab = "status",
+}: Props) => {
+  const [tabMode, setTabMode] = useState<TabMode>(initialTab)
 
   return (
     <div className="c-vehicle-properties-panel">
       <Header
         vehicle={selectedVehicle}
         tabMode={tabMode}
-        setTabMode={setTabMode}
+        setTabMode={(tab) => {
+          if (tab !== tabMode) {
+            window.FS?.event("Switched tab in Vehicle Properties Panel", {
+              tab_str: tab,
+            })
```

Comment:
question (non-blocking): Is there a sample event in FS to see? I'm sure how what it looks like in the FS UI when event details are passed like this, but as long as it is usable for Product/Design than LGTM!

Reply:
Good question! I haven't deployed to a dev env to check yet, but I will do so before merging. Do you remember which dev env is connected to FS?(IIRC Its dev blue?)

(Also, I'm excited for the deploy PR "from labels" work you did, and have been looking forward to using it 😃)

Reply:
Ah yeah I think you're right it is dev-blue. Just removed that label since I'm through testing with it, so feel free to deploy there! Looks like if we're ever in a pinch we could [temporarily turn on dev-green](https://app.fullstory.com/ui/AQKVD/settings/capture) if needed, though I'm not really sure if there would be FS quota restrictions to consider.

 All credit to the Glides team for writing that label => deploy action 😃

Reply:
The FS events are visible here! https://app.fullstory.com/ui/AQKVD/segments/everyone/people:search:kJDRqUfulAOm?completeSessions=false